src/components/MessagesList/MessagesList.vue

```diff
@@ -613,36 +615,36 @@ export default {
 			let isFocused = null
 			if (focusMessageId) {
 				// scroll to message in URL anchor
-				isFocused = this.focusMessage(focusMessageId, false)
+				return this.focusMessage(focusMessageId)
```

Comment:
returned value would be unexpected here
```suggestion
				this.focusMessage(focusMessageId)
				return
```